bcipy/language/tests/test_gpt2.py

```diff
@@ -0,0 +1,76 @@
+"""Tests for GPT2 Language Model"""
+
+import pytest
+import unittest
+from operator import itemgetter
+
+from bcipy.helpers.exceptions import UnsupportedResponseType
+from bcipy.helpers.task import alphabet
+from bcipy.language.model.gpt2 import GPT2LanguageModel, ResponseType
+
+
+@pytest.mark.slow
```

Comment:
One thing to note is that I marked this test suite as slow due to the load times of GPT2. If we can figure out a way to speed this up, without including the entire thing in our repository, we could remove this annotation.

Reply:
I'm okay with this for now! At some point, it would be nice to abstract these libraries from our unit tests and mock them.